content/07.pathogenesis.md

```diff
@@ -134,3 +134,41 @@ The network analysis showed in particular that non-structural proteins 9 and 10
 Finally, it was suggested [@doi:10.1101/2020.04.09.033522] that protein E of both SARS-CoV and SARS-CoV-2 has a conserved Bcl-2 Homology 3 (BH3)-like motif, that could inhibit anti-apoptosis proteins BCL2 and trigger the apoptosis of T cells.
 Several known compounds were suggested to disrupt the host-pathogen protein interactome were suggested mostly through the inhibition of host proteins.
+
+#### Transcriptomics of COVID-19
+
+To quickly associate the clinical outcomes with SARS-CoV-2 infection, researchers are taking advantage of “omics” technologies to profile the expression of coronavirus entry factors across the tissues/cells and also to measure the host transcriptional response after virus infection. 
+Although we make an effort to evaluate all relevant studies, we realize that this area is rapidly evolving.
+
+Two studies profiled expression response after SARS-CoV-2 infection in multiple human cell lines (Figure {@fig:cell-lines-moi}A) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. 
+The first study, which is by Blanco-Melo et al., performed an in-depth analysis of the transcriptional response to SARS-CoV-2 in comparison to other respiratory viruses, including MERS-CoV, SARS-CoV-1, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV). 
+They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
+As the viral receptor ACE2 has low expression in A549 cells, authors also supplemented A549 cells with a vector expressing mCherry or ACE2 (A549-ACE2). 
+All infections were performed at a high multiplicity of infections (MOI, 2–5), except A549-ACE2 cells were infected with SARS-CoV-2 at both low (0.2 MOI) and high MOI (2 MOI).
+Poly(A) bulk RNA sequencing (RNA-seq) of host cells was performed after 24 hours post-infection (hpi) of all the viruses except IAV (9-12 hours).
+The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes of ferrets as an animal model, and lung samples from two COVID-19 patients. 
+Additionally, transcriptional response to IAV in NHBE cells and nasal washes of ferrets was measured.
+Differential expression (DE) analysis was carried out between infected and respective mock (control) cell condition.
+DE results revealed that the transcriptional response of the cells with efficient SARS-CoV-2 replication is significantly different from the host response of all other viruses tested. 
+A unique proinflammatory cytokine signature is present in both high and low multiplicity of infections (MOI). 
+However, in contrast to high-MOI, low-MOI SARS-CoV-2-infected A549-ACE2 cells showed no significant IFN-I or IFN-III expression, suggesting that IFN induction is MOI dependent. 
```

Comment:
```suggestion
However, A549-ACE2 cells in the low-MOI condition showed no significant IFN-I or IFN-III expression, but cells at high-MOI did.
This finding suggested that IFN induction is MOI dependent. 
```
I tried to space it out a little to make it easier to follow, since there are lots of acronyms!
Could you possibly add a sentence saying why we care about IFN expression levels? Are these two IFN in particular important?